.changeset/gold-taxis-heal.md

```diff
@@ -0,0 +1,31 @@
+---
+"@comet/cms-admin": minor
+"@comet/admin": minor
+---
+
+Add `filterByDocument` to replace graphql-anywhere's `filter`
```

Comment:
alternative names:

- filterByFragment
- filterFragment
- fragmentFilter

Reply:
In theory it's possible to filter by query as well, but will we ever use that? `filterByFragment` is fine for me.

Reply:
Renamed to filterByFragment: 497e6a52520d9e5f93ec75ac5e5208bfaacd4df1